libs/vkd3d-shader/spirv.c

```diff
@@ -5766,9 +5769,12 @@ static void vkd3d_dxbc_compiler_emit_resource_declaration(struct vkd3d_dxbc_comp
 
         if (counter_binding && (counter_binding->flags & VKD3D_SHADER_BINDING_FLAG_BINDLESS))
         {
+            SpvStorageClass storage_class = (counter_binding->flags & VKD3D_SHADER_BINDING_FLAG_RAW_VA)
+                    ? SpvStorageClassUniform : SpvStorageClassUniformConstant;
```

Comment:
Why not StorageClassStorageBuffer? Is it because we're targeting SPIR-V 1.0 here?

Reply:
It's consistent with what we're doing elsewhere; moving to "proper" 1.3 (which we should probably do at some point) requires more work.

Reply:
Ok, no worries in that case.